source/js/text-ellipsis.js

```diff
@@ -1,14 +1,90 @@
-/*** 
- * @summary Sets the ellipsis based on the cpl - 3.
- * @param {Object} textLine: Node element, where to set the ellipsis
- * @param {Integer} characterPerLine: Characters per line. Set or calculate cpl.
+var cpl = require("./characters-per-line");
```

Comment:
maybe we can use a more readable name? later we can easily forgot what 'cpl' means.

Reply:
❓
The imports didn't work with
/* global CharactersPerLine:false */